threedigrid_builder/grid/connection_nodes.py

```diff
@@ -93,6 +91,14 @@ def is_closed(self, content_pk):
         """
         return self.storage_area[self.id_to_index(content_pk)] >= 0
 
+    def is_channel(self, content_pk, channels):
+        """Whether object is connected to a channel"""
+        has_channel = np.logical_or(
+            np.isin(self.id, channels.connection_node_start_id),
```

Comment:
Ik heb nog wel vraagteken bij deze check. Wat nu als de connection_node_start_id ook aan een pipe zit? Dan is dat nu altijd een channel, ondanks dat het misschien een manhole is en is_closed logischer is. Ik overzie het niet helemaal, maar potentieel is dit te strict.

Reply:
Dit lijkt me niet de plek om zoiets aan te passen; dit doet namelijk niet meer dan checken of iets een channel is. Als dat de lading niet dekt moet hooguit de naam worden aangepast.

> Ik overzie het niet helemaal, maar potentieel is dit te strict. Als kcu bekend is hier voor aanliggende 1d2d verbinding zou ik die checken, ook omdat de dpumax uiteindelijk gezet wordt voor de 1d2d line.

Lijkt me iets wat aangepast moet worden in het kiezen tussen `is_closed` en `is_channel`: https://github.com/nens/threedigrid-builder/blob/margriet_389_schema_300_1D/threedigrid_builder/grid/grid.py#L757-L765. Maar ik snap nog niet helemaal welke logica je hier zou willen zien. Is dat iets als:

```
if node_open_water_detection == 0 and isinstance(objects, ConnectionNodes):
  if line.kcu != -9999:
      is_closed = (line.kcu == LINE_1D2D_SINGLE_CONNECTED_CLOSED or line.kcu == INE_1D2D_DOUBLE_CONNECTED_CLOSED)
  else:
      is_closed = is_channel()
```